apps/rod/behaviours/custom-validations.js

```diff
@@ -0,0 +1,33 @@
+const validators = require('hof/controller/validation/validators');
+
+module.exports = superclass =>
+  class extends superclass {
+    validateField(key, req) {
+      const validationErrorFunc = (type, args) =>
+        new this.ValidationError(key, { type: type, arguments: [args] });
+        // eslint-disable-next-line max-len
+      if (req.form.values['rod-reference-number'].includes('ho-reference-number') && key === 'rod-ho-reference-number') {
```

Comment:
we can move the if condition to multiline so don't have to disable the rule

Reply:
Done :)